scripts/platformCheck.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("fs");
+const path = require("path");
+
+function getTargetName() {
+    switch (process.platform) {
+        case "darwin":
+            return `darwin-${process.arch}`;
+        case "linux":
+            const isMusl = process.report.getReport().header.glibcVersionRuntime == null;
+            switch (process.arch) {
+                case "arm":
+                    return "linux-arm-gnueabihf";
+                case "arm64":
+                    return `linux-arm64-${isMusl ? "musl" : "gnu"}`;
+                case "ia32":
+                    return "linux-ia32-gnu";
```

Comment:
I think we'll need to make a fix for `index.js` because currently it doesn't handle loading the x86 Linux binary